sort/gnome.go

```diff
@@ -0,0 +1,17 @@
+package sort
+
```

Comment:
Please document the function. Check out other sorting algorithms for reference.

Reply:
Done